content/en/docs/instrumentation/php/resources.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Resources
+weight: 6
+---
+
+A resource represents the entity producing telemetry as resource attributes. For
+example, a process producing telemetry that is running in a container on
+Kubernetes has a Pod name, a namespace, and possibly a deployment name. All
+three of these attributes can be included in the resource.
+
+In your observability backend, you can use resource information to better
+investigate interesting behavior. For example, if your trace or metrics data
+indicate latency in your system, you can narrow it down to a specific container,
+pod, or Kubernetes deployment.
+
+Below you will find some introductions on how to set up resource detection with
+the PHP SDK.
+
+## Resource Detection
+
+The PHP SDK detects resources from a variety of sources:
```

Comment:
Does it detect them by default, or do you need to tell it to detect this info?

Reply:
by default, and I've made it clearer that this happens as well as how to control which detectors are used.